docs/protocol.md

```diff
@@ -381,3 +381,79 @@
     <td colspan="2"><code>$ MOVE Main Savings 150.15</code><br /><code>SUCCESS</code></td>
   </tr>
 </table>
+
+<h2>Micro-loans</h2>
+
+<h3>REQUESTLOAN</h3>
+
+<table>
+  <tr>
+    <td><strong>Description</strong></td>
+    <td colspan="2">Request a peer-to-peer loan. Requests can be seen by all other users.</td>
```

Comment:
@ZacTredger "Requests can be seen by all other users." why is it so?

Reply:
Ah, it's coming from the requirements about P2P. I see now, sorry.

Reply:
Haha it's a bit misleading actually, because at the moment they can't actually be _seen_ by _any_ users... because I haven't implemented that.